src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,10 @@
+UART Device Emulation Module (DEM_UART)
+=======================================
+
+TODO: Add diagram showing internal structure?
+
+The UART Device Emulation Module (DEM_UART) connects to the bus of a given CPU on one side,
+and to the Debug Interconnect Interface on the other.
+
+Towards the CPU it behaves like a UART-16550 device, but instead of transmitting information
+over a UART-Interface it instead passes it to a host PC via the Debug Interconnect Interface.
```

Comment:
Somewhere you need to 
- describe the CPU interface in more detail
- discuss which features of which version of the UART 16550 device you support (and which ones are not supported)

You can additionally link to a data sheet or other online resources of a device that your implementation is compatible with.